### DISCUSSION TITLE
Accordion and Pane IB

### DISCUSSION THREAD

**Discussion by Olekrut** [maxi-blocks/maxi-blocks#5289] [Announcements]
1) Accordion: Add to IB list 
![image](https://user-images.githubusercontent.com/88893350/193376700-083561d7-f2bc-4cfb-825a-6b8e7d79106b.png)
2) Pane: Add to IB list
![image](https://user-images.githubusercontent.com/88893350/193376654-1a9929f6-9228-4f5a-947a-820fdff4a038.png)

**Comment by kyrapieterse** [comment]
Do we need this? can't think of a use case 🤔
